web/pages/base_page.rb

```diff
@@ -4,6 +4,35 @@ module Web
   module Pages
     class BasePage <SitePrism::Page
       element :logo, 'a.header__logo'
+      element :search_field, 'input.search-form__input'
+      element :search_button, 'button.search-form__submit'
+      element :clear_button, 'button.search-form__clear'
+      elements :suggestion_items, 'ul.suggest-list li.search-suggest__item'
+      element :empty_results, "div.search-nothing div h1"
```

Comment:
if this element contains text, please name it appropriately. For example:

```suggestion
      element :empty_results_text, "div.search-nothing div h1"
```